pandas/core/frame.py

```diff
@@ -5432,7 +5433,7 @@ def _put_str(s, space):
 
 import pandas.tools.plotting as gfx
 
-DataFrame.plot = gfx.plot_frame
+DataFrame.plot = base.AccessorProperty(gfx.FramePlotMethods, gfx.FramePlotMethods)
 DataFrame.hist = gfx.hist_frame
```

Comment:
hmm, maybe we should deprecate ``DataFrame.hist`` (and point to ``DataFrame.plot.hist``) as the canonical way?